qtox.pro

```diff
@@ -35,7 +35,8 @@ FORMS    += \
     src/widget/form/settings/generalsettings.ui \
     src/widget/form/settings/privacysettings.ui \
     src/widget/form/removefrienddialog.ui \
-    src/widget/about/aboutuser.ui
+    src/widget/about/aboutuser.ui \
+    src/widget/form/settings/userinterfacesettings.ui
```

Comment:
Not urgent, but would be nice to move it up to "settings block". Same with .cpp and .h entries.